src/data/builds/shaman/2001_2_Alchemy_Choppin_hybrid.ts

```diff
@@ -4,7 +4,7 @@ import { Build } from "~/composables/Builds";
 
 export const build: Build = {
   title: "Alchemy / Choppin Hybrid (Shaman) 100+",
-  notes: "",
+  notes: " ",
```

Comment:
👀

Reply:
I believe this prevents seeing "Notes about your perfect build. Explain the order of upgrades, tradeoffs, or other important notes.

Reply:
Ah I guess there's just no build description then because that's the default filler text.